custom_components/dobiss/light.py

```diff
@@ -42,9 +42,9 @@ def __init__(self, coordinator, light):
     @property
     def supported_features(self):
         if self.dobiss.modules[self._light['moduleAddress']]['type'] == DobissSystem.ModuleType.Relais:
-            return (SUPPORT_FLASH | SUPPORT_TRANSITION)
+            return LightEntityFeature.FLASH | LightEntityFeature.TRANSITION | SUPPORT_BRIGHTNESS
```

Comment:
This is for relais, so no brightness support, right?

I wrote this code years ago, so maybe by now there is a device type that is a better fit.

Reply:
yeah, you're right